platform/datacollector/src/main.rs

```diff
@@ -14,7 +19,7 @@ async fn health_handler() -> impl Responder {
 #[get("/data")]
```

Comment:
if there's just a single endpoint, I would just make it root `/`, easier for the knative (you can have multiple endpoints though, not a big deal, just so far that's what seems to be the idiom)